.editorconfig

```diff
@@ -0,0 +1,83 @@
+# ***************************
```

Comment:
FWIW, we haven't put any `.editorconfig` files in our frontend repos... I'm not sure it's necessary.  I'm fairly sure our eslint config accounts for all this.